docs/.dockerignore

```diff
@@ -0,0 +1,12 @@
+.git/
+.github/
+build/
+.editorconfig
+.gitattributes
+.gitignore
+CHANGELOG.md
+CODE_OF_CONDUCT.md
+deploy.sh
+font-selection.json
+README.md
+Vagrantfile
```

Comment:
End of file here. Maybe running the pre-commit will help cleaning up these small things